snuba/migrate.py

```diff
@@ -15,7 +15,7 @@ def _run_schema(conn, schema):
         return
     clickhouse_table = schema.get_local_table_name()
     get_schema = lambda: {
-        column_name: column_type
+        column_name: (column_type, default_type)
```

Comment:
Following up on the comment in `snuba/datasets/events.py` — it seems like `column_type, default_type, default_expr` all should just be packed into a `NamedTuple` or something.